signer/src/service/mod.rs

```diff
@@ -24,8 +24,19 @@ pub fn create_account() -> (Mnemonic, AccountInfo) {
     (mnemonic, account_info)
 }
 
-pub fn get_account(mnemonic: &str) -> Result<AccountInfo> {
+pub fn get_account_by_mnemonic(mnemonic: &str) -> Result<AccountInfo> {
     let mnemonic = Mnemonic::from_phrase(mnemonic, Language::English)?;
+    get_account(mnemonic)
+}
+
+pub fn get_account_by_bip39_entropy(bip39_entropy: &str) -> Result<AccountInfo> {
+    let mut entropy = [0u8; 32];
+    hex::decode_to_slice(bip39_entropy, &mut entropy)?;
```

Comment:
🤦 I've been decoding to a vec and then using `try_into()`, this is much more concise and less likely for someone to use `clone_from_slice` which panics if the sizes are wrong.